opennms-dao/pom.xml

```diff
@@ -114,7 +114,6 @@
     <dependency>
       <groupId>org.opennms.features.collection</groupId>
       <artifactId>org.opennms.features.collection.persistence.rrd</artifactId>
-      <scope>test</scope>
```

Comment:
I've done that first, but it looks strange because it's only related to the filesystem implementation. I will have a look if I can find a better place to do this.